pkg/management/http.go

```diff
@@ -69,11 +70,14 @@ func managementErrorHandler(logger log.Logger) func(context.Context, error, http
 		switch e := errors.Cause(err).(type) {
 		case kc_client.HTTPError:
 			w.WriteHeader(e.HTTPStatus)
+			w.Write([]byte(internal.ComponentName + "." + internal.MsgErrUnknown))
 		case ConvertLocationError:
 			// 201-Created, even if ConvertLocationError occurs, the creation was a success
 			w.WriteHeader(http.StatusCreated)
+			w.Write([]byte(internal.ComponentName + "." + internal.MsgErrUnknown))
```

Comment:
I don't think we should return an error message with HTTP 201 status code

Reply:
Thanks for the comment: this has been updated